tla-bmcmt/src/test/scala/at/forsyte/apalache/tla/bmcmt/TestSeqModelCheckerTrait.scala

```diff
@@ -44,7 +43,7 @@ trait TestSeqModelCheckerTrait extends FixtureAnyFunSuite {
     val inv = not(notInv).typed(types, "b")
     val checkerInput =
       new CheckerInput(mkModuleWithX(), initTrans, nextTrans, None, CheckerInputVC(List((inv, notInv))))
-    val params = new ModelCheckerParams(checkerInput, stepsBound = 0, new File("."), Map(), false)
+    val params = new ModelCheckerParams(checkerInput, stepsBound = 0, Map(), false)
```

Comment:
Same as above, there's a lot of repetition here that could be put in a method.

Reply:
Yeah, but that's an existing test, I merely removed an unused parameter here in [dfe6072](https://github.com/informalsystems/apalache/pull/1635/commits/dfe6072f3f957973c0acd90c9b4d4bae74972b84).
Happy to file a separate issue to clean this up (tho I'm not super concerned about duplication in existing unit tests).